src/fr/reaperscans/src/eu/kanade/tachiyomi/extension/fr/reaperscans/ReaperScans.kt

```diff
@@ -1,13 +1,33 @@
 package eu.kanade.tachiyomi.extension.fr.reaperscans
 
 import eu.kanade.tachiyomi.multisrc.keyoapp.Keyoapp
+import eu.kanade.tachiyomi.source.model.SManga
+import org.jsoup.nodes.Document
 
 class ReaperScans : Keyoapp(
     "Reaper Scans",
     "https://reaper-scans.fr",
     "fr",
 ) {
 
+    override fun mangaDetailsParse(document: Document): SManga =
+        super.mangaDetailsParse(document).apply {
+            // Fix a typo in the class name
+            if (description.isNullOrBlank() == true) {
+                description = document.selectFirst("div.grid > div.overflow-hiddfen > p")?.text()
+            }
+
+            // Check that text is "Statut"
```

Comment:
Typo

Reply:
This is wanted. On some chapters, there is this typo. That why this is conditional.